tests/example.config.pyyaml-include.yaml

```diff
@@ -0,0 +1,21 @@
+rules_folder: /opt/elastalert/rules
+
+run_every:
+  seconds: 10
+
+buffer_time:
+  minutes: 15
+
+es_host: elasticsearch
+es_port: 9200
+
+es_username: elastic
+es_password: $ELASTIC_PASS
+
+writeback_index: elastalert_status
+
+alert_time_limit:
+  days: 2
+
+include_from_file_seq: !include /home/elastalert/tests/example.config.pyyaml-include.target.seq.yaml
+include_from_file_map: !include /home/elastalert/tests/example.config.pyyaml-include.target.map.yaml
```

Comment:
Specifying path like this make the test dependent on tests/docker-compose.yml and tests/Dockerfile-test.

I could create a new yaml dynamically on runtime to have the right path under different test environments using os.path.dirname(os.path.realpath(__file__)) and delete the file after the test, but not sure that is a better way to do this.

Reply:
Can it use relative paths instead of absolute? On that same topic I see that you're using a base_dir of `/opt/elastalert` in the yaml.py file, but non-Docker users might not use that same absolute path. Could it be left blank to perhaps assume current working dir? I haven't looked into the yaml module yet.